git_machete/exceptions.py

```diff
@@ -12,3 +12,7 @@ def __str__(self) -> str:
 class StopTraversal(Exception):
     def __init__(self) -> None:
         pass
+
+
+class UnprocessableEntityError(Exception):
```

Comment:
I'd specify:
```suggestion
class UnprocessableHttpEntityError(Exception):
```

Reply:
I would suggest rather not splitting Unprocessable - Entity and Http - Error pairs since when coupled together they have special meaning. So either UnprocessableEntityHttpError or HttpErrorUnprocessableEntity.